Dockerfile.rocm

```diff
@@ -197,6 +227,14 @@ RUN --mount=type=bind,from=export_amdsmi,src=/,target=/install \
     pip uninstall -y amdsmi \
     && pip install /install/*.whl;
 
+RUN --mount=type=bind,from=export_pytorch,src=/,target=/install \
+    if ls /install/*.whl; then \
+        # Preemptively uninstall to prevent pip same-version no-installs
+        pip uninstall -y torch torchvision \
+        && pip install /install/*.whl \
+        && python3 -m pip install --no-cache-dir --pre torchvision --index-url https://download.pytorch.org/whl/nightly/rocm6.2; \
```

Comment:
We don't need this if we're building `torchvision` above?

Reply:
Oh a second point is we should install the `torch` wheel first before all others: `torch` might in some cases bundle `pytorch-triton-rocm` which would overwrite the `triton` installation above.

Reply:
This was a leftover from an attempt to use whls. No whls for us as this one brings torch whl, which brings all the things we don't want